Documentation/tutorial/validate your markdown files.md

```diff
@@ -0,0 +1,128 @@
+# Validate your markdown files
+
+In GFM, we can write any document with right syntax. For example, we can write html tag `<h1>title</h1>` instead of markdown syntax `#title`. But for some purpose, some behaviors are unwanted, so we add markdown validators for each markdown document.
+
+In this document, you'll learn how to define markdown validation rules, which will help you to validate markdown documents in an efficient way.
+
+*This is part of DFM, if you switch markdown engine to other engine, validation might not work.*
+
+## How to config markdown validation rules
+
+Create a *JSON* file with name `md.style` in the same folder of `docfx.json`, and with following content:
+```json
+{
+   "rules": [],
+   "tagRules": []
+}
+```
+
+## Build-in html tag validation rules
+
+For most cases, we might need some html tag rules, and we have a build-in tag rules.
```

Comment:
How about:
For most cases, you may want to prohibit using certain html tags in markdown, so we built a built-in html tag rule for you.